releasenotes/notes/scrub-log-output-b66be1ad3cf26313.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
```

Comment:
To address before merging: I think we should list this under `other`: release notes reflect user-facing changes, and this PR shouldn't change much to the user-facing behavior of the agent. In the same spirit, we could reword the message below to something like `should prevent leakage of credentials in logs from 3rd-party code or code added in the future`